docs/roadmap.md

```diff
@@ -0,0 +1,282 @@
+believe to be on a good path.  The iOS app has fewer features than
+Android and has more bugs, but more importantly is in need of an
+experienced iOS developer who can drive the project.
+
+## Core User Experience
+
+This category includes important improvements to the core user
+experience that will benefit all users.
+
+* [Improve missed message notifications to make "reply" work nicely](https://github.com/zulip/zulip/issues/612)
+* [Add support for showing "user is typing" notifications](https://github.com/zulip/zulip/issues/150)
+* [Add pretty bubbles for recipients in the compose box](https://github.com/zulip/zulip/issues/595)
+* [Finish and merge support for pinning a few important streams](https://github.com/zulip/zulip/issues/285)
+* [Display stream descriptions more prominently](https://github.com/zulip/zulip/issues/164)
+* [Integration inline URL previews](https://github.com/zulip/zulip/issues/406)
+* [Add support for managing uploaded files](https://github.com/zulip/zulip/issues/454)
+* [Make Zulip onboarding experience smoother for teams not used to topics](https://github.com/zulip/zulip/issues/647).  That specific proposal might not be right but the issue is worth investing time in.
+
+## Ease of setup and onboarding issues
+
+This category focuses on issues users experience when installing a new
+Zulip server or setting up a new Zulip realm.
+
+* [Create a web flow for setting up a new realm / the first realm on a new server (currently, it's a command-line process)](https://github.com/zulip/zulip/issues/260)
+* [Document or better script solution to rabbitmq startup issues](https://github.com/zulip/zulip/issues/465)
+* [Add a mechanism for deleting early test messages](https://github.com/zulip/zulip/issues/135)
+* [Merge a supported way to use Zulip in Docker in production
+  implementation](https://github.com/zulip/zulip/pull/450).
+
+## Internationalization
+
+The core Zulip UI has been mostly translated into 5 languages;
+however, more work is required to make those translations actually
+displayed in the Zulip UI for the users who would benefit from them.
+
+* [Merge support for using translations in Django templates](https://github.com/zulip/zulip/pull/607)
+* Add text in handlebars templates to translatable string database
+* Add text in error messages to translatable string database
+* Merge support for translating text in handlebars
+* Merge support for translating text in error messages
+* Add tools for debugging internationalization issues (e.g. seeing what UI looks bad with longer words)
+
+## User Experience at scale
+
+There are a few parts of the Zulip UI which could benefit from
+overhauls designed around making the user experience nice for large
+teams.
+
+* [Make the buddy list work better for large teams](https://github.com/zulip/zulip/issues/236)
+* [Improve @-mentioning syntax based on stronger unique identifiers](https://github.com/zulip/zulip/issues/374)
```

Comment:
It would be nice to have possibility of making dynamic groups of people within stream that can be notified by simple @-mention, e.g @ops people within `devops` stream
I understand that this functionality can be implemented with different streams with right members, but I think large number of streams with flat structure is not very convenient to manage